src/Transition/OutputRulesFiles/OutputChanges9-2-0-to-9-3-0.md

```diff
@@ -12,3 +12,40 @@ This will eventually become a more structured file, but currently it isn't clear
 A new column "Peak Load Type to Size On" has been added to the `Coil Sizing Details:Coils` report
 
 See [7397](https://github.com/NREL/EnergyPlus/pull/7397)
+
+### Water to Water Air-Source EIR Heat Pump
+The HeatPump:WaterToWater:EIR:Heating and HeatPump:WaterToWater:EIR:Cooling objects have been renamed to HeatPump:PlantLoop:EIR:Heating and HeatPump:PlantLoop:EIR:Cooling, and the respective output variables have been renamed from: 
+
+*Before*
+```
+Water to Water Heat Pump Load Side Heat Transfer Rate
+Water to Water Heat Pump Load Side Heat Transfer Energy
+Water to Water Heat Pump Source Side Heat Transfer Rate
+Water to Water Heat Pump Source Side Heat Transfer Energy
+Water to Water Heat Pump Load Side Inlet Temperature
+Water to Water Heat Pump Load Side Outlet Temperature
+Water to Water Heat Pump Source Side Inlet Temperature
+Water to Water Heat Pump Source Side Outlet Temperature
+Water to Water Heat Pump Electric Power
+Water to Water Heat Pump Electric Energy
+Water to Water Heat Pump Load Side Mass Flow Rate
+Water to Water Heat Pump Source Side Mass Flow Rate
+```
+
+*9.3.0*
+```
+Plant Loop Heat Pump Load Side Heat Transfer Rate
+Plant Loop Heat Pump Load Side Heat Transfer Energy
```

Comment:
In general we try to have generic output variables names that are similar across different equipment (e.g. all chiller types have the same output variable names, except maybe a few oddballs). Other equipment, such as HeatPump:WaterToWater:EquationFit:Cooling use the old Water to Water Heat Pump * names here. But I see how that doesn't work when it's air-source. 

Maybe we want to consider changing the other equipment to match this one?  That would make the transition code easier. As it stands now, transition will change any occurrence of Water to Water Heat Pump * output variables to the new Plant Loop Heat Pump * names - it doesn't check to see what type of equipment is being referenced.

So, we either need to change these names everywhere, or we need to add smarts to transition to only change these for this type of equipment.

Reply:
Maybe we should just drop the "Plant Loop" *and* "Water to Water" on all of these outputs?

Reply:
That's a thought. How would that align with the other heat pump objects like heat pump water heater, water-to-air, and air-to-air?

Reply:
I think the answer to this will just come from what the RDD looks like in different cases.  Does unifying the variable names across different components make it easier on the user or more difficult.  The RDD doesn't give an obvious clue as to the linkage between components and output variables unless the component is actually in the variable name.  This might end up being a bigger discussion and so my recommendation is, whatever we do, not dwell too heavily on it for this particular PR.

At this moment, I'd probably support just making this "Heat Pump Load Side Foo Bar", and then maybe follow-up with a PR to unify the other heat pump objects with the same name, and evaluate the effects on the RDD over in that new PR instead of this one.

Reply:
Let's see, these following take the "Load Side" - "Source Side" approach: 

```HeatPumpWaterToWaterCOOLING.cc```
"Water to Water Heat Pump Electric Power"
"Water to Water Heat Pump Load Side Heat Transfer Rate"

```HeatPumpWaterToWaterHEATING.cc```
"Water to Water Heat Pump Electric Power"
"Water to Water Heat Pump Load Side Heat Transfer Rate"

```HeatPumpWaterToWaterSimple.cc```
"Water to Water Heat Pump Electric Power"
"Water to Water Heat Pump Load Side Heat Transfer Rate"

```PlantLoopHeatPump.cc```
"Plant Loop Heat Pump Load Side Heat Transfer Rate"
"Plant Loop Heat Pump Source Side Heat Transfer Rate

```WaterToAirHeatPump.cc```
"Heating Coil Electric Energy"
"Heating Coil Source Side Heat Transfer Energy"


This goes with specifying "Air" - "Water":

```IntegratedHeatPump.cc```
"Integrated Heat Pump Electric Power"
"Integrated Heat Pump Water Heating Rate"
"Integrated Heat Pump Air Latent Cooling Rate"


This mixes "Source Side" and "Latent/Sensible":

```WaterToAirHeatPumpSimple.cc```
"Cooling Coil Electric Energy"
"Cooling Coil Source Side Heat Transfer Energy"
"Cooling Coil Latent Cooling Energy"


This looks like it doesn't follow any convention:

```WaterThermalTanks.cc -- WaterHeater:HeatPump:PumpedCondenser```
"Water Heater On Cycle Ancillary Electric Power"
"Water Heater Heat Pump Control Tank Temperature"


Are these included?

```PackagedTerminalHeatPump.cc```
"Zone Packaged Terminal Heat Pump Electric Power"
"Zone Packaged Terminal Heat Pump Latent Cooling Rate"

```HVACMultiSpeedHeatPump.cc```
"Unitary System Electric Power"

In summary, it looks like there could be some standardization of output vars, but maybe we handle that separately.

Reply:
There is a difference between parent and child power/energy report variable names. It is the child that reports to the meter and the parent only echoes this data as a convenience for the user (and does not report to a meter). So when choosing report names, it is good to use the same name for say coils, Cooling Coil Total Cooling Energy, because the energy use from all coil types can be aggregated by using a single name. For parent objects this is not all that important, unless of course you want to aggregate total power/energy for systems, because the parent object is not responsible for meter reporting. For example, if Unitary System Total Cooling Energy/Rate were used for all system types (PTHP, UnitarySystem, Furnace, etc.) then a single report name could be used to report or aggregate all the information (coils and fans) in the entire simulation but would not discern between coils and fans or need to match specific meters. This is most important for UIs where they don't need to keep track of different report variable names for different system types. And for UIs I think what is important is that the magnitude of the report data matches what the meter would report. So for parent object, it would be nice to be able to use a single name to get reports, but it's not as important as for child components.

Take a report like this for example, it's more important when there are child water coils and DX coils in the same simulation to use the same name. You would not want to do this twice. For parent objects, this is not as important, but we need to think this out in case it makes sense to use the same logic for parents.

    Output:Table:Monthly,
    Coil Report,                              !- Name
    ,                                       !- Digits After Decimal
    Cooling Coil Total Cooling Energy,      !- Variable or Meter Name 1
    SumOrAverage,                           !- Aggregation Type for Variable or Meter 1
    Cooling Coil Electric Energy,           !- Variable or Meter Name 2
    SumOrAverage,                           !- Aggregation Type for Variable or Meter 2
    Cooling Coil Total Cooling Energy,      !- Variable or Meter Name 3
    HoursNonZero,                           !- Aggregation Type for Variable or Meter 3
    Cooling Coil Sensible Cooling Energy,   !- Variable or Meter Name 4
    SumOrAverage,                           !- Aggregation Type for Variable or Meter 4
    Cooling Coil Latent Cooling Energy,     !- Variable or Meter Name 5